db/migrate/20180531063202_add_bankaccount_to_person.rb

```diff
@@ -0,0 +1,5 @@
+class AddBankaccountToPerson < ActiveRecord::Migration[5.0]
+  def change
+    add_column :people, :bankaccount, :string, null: true
```

Comment:
Waarschijnlijk zie ik iets over het hoofd, maar waar wordt dit bankaccount attribuut gebruikt? Ik zie alleen dat `iban` attribuut terugkomen in de forms en de validators enzo.